lib/app/modules/hotel_details/views/components/hotel_details_exploring.dart

```diff
@@ -0,0 +1,163 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:rihlatic/app/core/components/cards/tag_component.dart';
+import 'package:rihlatic/app/core/constants/icons_assets_constants.dart';
+import 'package:rihlatic/app/core/constants/strings_assets_constants.dart';
+import 'package:rihlatic/app/core/styles/main_colors.dart';
+import 'package:rihlatic/app/core/styles/text_styles.dart';
+
+class HotelDetailsExploring extends StatelessWidget {
+  const HotelDetailsExploring({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 20.0),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text(
+            'Exploring the area',
+            style: TextStyles.mediumLabelTextStyle(context).copyWith(
+              color: MainColors.textColor(context)!,
+            ),
+            maxLines: 2,
+            textAlign: TextAlign.left,
+          ),
+          SizedBox(
+            height: 10.h,
+          ),
+          Row(
```

Comment:
change structure of this section like this:

Column 
    Row
       Row Row
    Row
       Row Row